Assets/Scripts/MoveTrack.cs

```diff
@@ -22,11 +22,11 @@ public enum Type
     }
    [SerializeField] private Way arrow = Way.VERT ;
    [SerializeField] public  Type path = Type.ROTATE ; 
-    public float perUnit = 0.05f  ; 
+    public float perUnit = 1f  ; 
     private float rotate ; 
     void Start()
     {
-        perUnit = perUnit * intenstity ; 
+        perUnit = perUnit * intenstity * Time.deltaTime; 

```

Comment:
Time.deltaTime is the time since the last rendered frame. I don't think it makes sense being used here in start().